sys-apps/ripgrep/ripgrep-0.10.0.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Combines the usability of ag with the raw speed of grep"
+DESCRIPTION="ripgrep is a line-oriented search tool that recursively searches \
+your current directory for a regex pattern while respecting your gitignore \
+rules. It is similar to other popular search tools like The Silver Searcher, \
+ack and grep."
+HOMEPAGE="https://github.com/BurntSushi/ripgrep"
+COPYRIGHT="2015 Andrew Gallant"
+LICENSE="Unlicense
+	MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="a2a6eb7d33d75e64613c158e1ae450899b437e37f1bfbd54f713b011cd8cc31e"
+SOURCE_FILENAME="ripgrep-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+PROVIDES="
+	ripgrep$secondaryArchSuffix = $portVersion
+	cmd:rg$commandSuffix
```

Comment:
This recipe does not provide any library so we can get rid of the `commandSuffix` with these changes:
```diff
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+fi
 
-commandSuffix=$secondaryArchSuffix
-if [ "$targetArchitecture" = x86_gcc2 ]; then
-	commandSuffix=
-fi
 
 PROVIDES="
 	ripgrep$secondaryArchSuffix = $portVersion
-	cmd:rg$commandSuffix
+	cmd:rg
 	"
```
(If you copy/paste the if then fi block, make sure you get rid of the white space before `SECONDARY_ARCHITECTURES="x86"` :)

Reply:
Good catch! The guide does need a small update... I'll fix it soon :)
What changed is that with the conditional secondary arch the commandSuffix var is now always empty, which means we can drop it.

Reply:
Wiki updated :tada:
https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines/_compare/bbcb32a%5E...bbcb32a